### PR TITLE
[com_fields] Show categories when a category is available in fields list

### PR DESCRIPTION
**Pull request by laoneo**
Pull Request for Issue #14373.

### Summary of Changes
Don't show the category information in the fields list, when the component doesn't use categories.

### Testing Instructions
- Log in on the back end
- Go to Users -> Fields
- Create a new custom field

### Expected result
Below the custom field title is no label "Category: All" shown.

### Actual result
"Category: All" label is shown below the field title.

### CODE REVIEW

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 3aa9165d16335cea2cc8f6e29c71ceaafa33f5f4<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14405">issues.joomla.org/tracker/joomla-cms/14405</a>.</sub>

File: administrator/components/com_fields/views/fields/tmpl/default.php [hunk 1]
```diff
@@ -142,11 +145,13 @@
...
 									</span>
 									<div class="small">
-										<?php echo JText::_('JCATEGORY') . ': '; ?>
-										<?php if ($categories = FieldsHelper::getAssignedCategoriesTitles($item->id)) : ?>
-											<?php echo implode(', ', $categories); ?>
-										<?php else: ?>
-											<?php echo JText::_('JALL'); ?>
+										<?php if ($category) : ?>
+											<?php echo JText::_('JCATEGORY') . ': '; ?>
+											<?php if ($categories = FieldsHelper::getAssignedCategoriesTitles($item->id)) : ?>
```
Comment (zero-24) [inline administrator/components/com_fields/views/fields/tmpl/default.php]: can we change this to:

```php
<?php $categories = FieldsHelper::getAssignedCategoriesTitles($item->id); ?>
<?php if ($categories) : ?>
```

Comment (laoneo) [inline administrator/components/com_fields/views/fields/tmpl/default.php]: go for it, when you want

**Comment by coolcat-creations** [comment]
I have tested this item :white_check_mark: successfully on 3aa9165d16335cea2cc8f6e29c71ceaafa33f5f4<br /><br />Works, thank you !!!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14405">issues.joomla.org/tracker/joomla-cms/14405</a>.</sub>

**Comment by zero-24** [comment]
@laoneo see: https://github.com/joomla/joomla-cms/pull/14405/commits/5e47caa7279b34afa2b3922d1642b5828923d086

**Comment by laoneo** [comment]
@zero-24 looks fine, RTC?

**Comment by zero-24** [comment]
I have not noticed the seccond test. :smile: RTC. Thanks to all :smile:

**Pull request merged**